src/test/scala/com/exasol/cloudetl/FileManager.scala

```diff
@@ -0,0 +1,42 @@
+package com.exasol.cloudetl
+
+import java.io.IOException
+import java.nio.file._
+import java.nio.file.attribute.BasicFileAttributes
+
+/**
+ * A mixin trait with file creation and deletion operations.
+ */
+trait FileManager {
```

Comment:
I think it's still nice to have Dummy(or something similar) in the class name, as it helps us to understand that it belongs to the test code

Reply:
And the same for the class above

Reply:
Done.